zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/AQLGenerator.scala

```diff
@@ -91,7 +108,7 @@ class AQLGenerator extends IQLGenerator {
       }
     }
     
-    val varMapAfterLookup = parseLookup(query.lookup, schemaVars, schemaMap)
+    val varMapAfterLookup = parseLookup(query.lookup, schemaVars, lookupSchemaMap)
```

Comment:
Actually not. I saw line 170 of `AQLGenerator` and this is the only place to use this schemaMap and actually this line is useless. I just doubt so I didn't change it. Can I remove line 170 in `AQLGenerator`?